pom.xml

```diff
@@ -31,12 +33,12 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>jackson2-api</artifactId>
-      <version>2.9.7.1</version>
+      <version>2.10.0</version>
     </dependency>
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>github-api</artifactId>
-      <version>1.95</version>
+      <version>1.99</version>
```

Comment:
Tip: `requireProperty` as in https://github.com/jenkinsci/incrementals-tools#superseding-maven-releases

Reply:
I'm sorry, what are you asking to be done here?

Reply:
Just a suggestion, not a request: that you can make sure `version` and `revision` do not drift out of synch by accident.